sdk/storage/azure-storage-blob/azure/storage/blob/_shared/encryption.py

```diff
@@ -115,23 +154,32 @@ def __init__(self, content_encryption_IV, encryption_agent, wrapped_content_key,
             A dict containing metadata related to the key wrapping.
         '''
 
-        _validate_not_none('content_encryption_IV', content_encryption_IV)
         _validate_not_none('encryption_agent', encryption_agent)
         _validate_not_none('wrapped_content_key', wrapped_content_key)
 
+        # Validate we have the right info for the specified algorithm
```

Comment:
Nit: Mostly no one is looking here anyways, but maybe we can put "matching required parameter"